docs/getting-started/tutorial-sample-data.asciidoc

```diff
@@ -1,206 +1,160 @@
-on the `kibana_sample_data_*` indices. See
-{ref}/security-privileges.html[Security privileges] for more information.
+* <<filter-and-query-the-data, Filter and query the dashboard data>>
 
+NOTE: If security is enabled, you must have `read`, `write`, and `manage` privileges
+on the `kibana_sample_data_*` indices. For more information, refer to
+{ref}/security-privileges.html[Security privileges].
 
 [float]
-=== Add sample data
+[[add-the-sample-data]]
+=== Add the sample data
 
-Install the Flights sample data set, if you haven't already.
+Add the *Sample flight data*.
 
 . On the home page, click *Load a data set and a {kib} dashboard*. 
+
 . On the *Sample flight data* card, click *Add data*.
-. Once the data is added, click *View data > Dashboard*.
-+
-You’re taken to the *Global Flight* dashboard, a collection of charts, graphs, 
-maps, and other visualizations of the the data in the `kibana_sample_data_flights` index.
-+
-[role="screenshot"]
-image::getting-started/images/tutorial-sample-dashboard.png[]
 
 [float]
-[[tutorial-sample-filter]]
-=== Filter and query the data
+[[explore-the-data]]
+=== Explore the data
 
-You can use filters and queries to
-narrow the view of the data. 
-For more detailed information on these actions, see 
-{ref}/query-filter-context.html[Query and filter context].
+Explore the documents in the index that 
+match the selected index pattern. The index pattern tells {kib} which {es} index you want to
+explore. 
 
-[float]
-==== Filter the data
+. Open the menu, then go to *Discover*.
 
-. In the *Controls* visualization, select an *Origin City* and a *Destination City*.
-. Click *Apply changes*.
+. Make sure `kibana_sample_data_flights` is the current index pattern. 
+You might need to click *New* in the {kib} toolbar to refresh the data.
 +
-The `OriginCityName` and the `DestCityName` fields filter the data on the dasbhoard to match 
-the data you specified. 
+You'll see a histogram that shows the distribution of 
+documents over time. A table lists the fields for 
+each document that matches the index. By default, all fields are shown.
 +
-For example, the following dashboard shows the data for flights from London to Milan. 
+[role="screenshot"]
+image::getting-started/images/tutorial-sample-discover1.png[]
+
+. Hover over the list of *Available fields*, then click *Add* next 
+to each field you want explore in the table.
 +
 [role="screenshot"]
-image::getting-started/images/tutorial-sample-filter.png[]
+image::getting-started/images/tutorial-sample-discover2.png[]
 
-. To add a filter manually, click *Add filter*,
-then specify the data you want to view.
+[float]
+[[view-and-analyze-the-data]]
+=== View and analyze the data
 
-. When you are finished experimenting, remove all filters.
+A _dashboard_ is a collection of panels that provide you with an overview of your data that you can 
+use to analyze your data. Panels contain everythng you need, including visualizations, 
```

Comment:
"...a collection of panels that provide you with an overview of your data that you can use to analyze your data" sounds a little stilted to me. 

Maybe something like "...a collection of panels that you can use to analyze your data"

Additionally, small typo here - should be "Panels contain everyth**i**ng you need"